common/protos/index.ts

```diff
@@ -24,7 +28,8 @@ export interface IProtoClass<IProto, Proto> {
 // meaning that the type of fields cannot be verified; an int can be confused with a string.
 export function verifyObjectMatchesProto<Proto>(
   protoType: IProtoClass<any, Proto>,
-  object: object
+  object: object,
+  options?: { suggestReportToDataformTeam?: boolean; showDocsLink?: boolean }
```

Comment:
sorry, now I see that it looks like these options are effectively an enumeration - it essentially never makes sense for them both to be true; because `suggestReportToDataformTeam` always takes precedence.

I'd suggest:

```
errorBehaviour?: enum ErrorBehaviour { SUGGEST_REPORTING_TO_DATAFORM_TEAM, SHOW_DOCS_LINK }
```